staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -227,29 +231,30 @@ type ConditionFunc func(info *resource.Info, o *WaitOptions) (finalObject runtim
 // RunWait runs the waiting logic
 func (o *WaitOptions) RunWait() error {
 	visitCount := 0
-	err := o.ResourceFinder.Do().Visit(func(info *resource.Info, err error) error {
-		if err != nil {
+	for {
```

Comment:
With low latency network this logic may penetrate the API server. May be we should limit rate here somehow?

Reply:
Now I'm waiting 1s before each interaction